content/en/blog/2023/kubecon-eu.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at KubeCon EU 2023
+linkTitle: KubeCon EU '23
+date: 2023-03-30
+spelling:
+  cSpell:ignore Severin Neumann Pavol Loffay Kowall Aiven Vider Xiaochun Alolita
+  Sharma Dyla Benedikt Bongartz Oliveira Pathak
+author: '[Severin Neumann](https://github.com/svrnm)'
+---
+
+The OpenTelemetry project maintainers, and members of the governance committee
+and technical committee are excited to be at [KubeCon EU][] in Amsterdam from
+April 18 - 21, 2023!
+
+You can meet us at the [project booth][], listen to our talks or learn with us
+during workshops. We are looking forward to hear from you, how you are using
+OpenTelemetry or contributing to the project.
+
+Below you will find a list of talks and other activities related to
+OpenTelemetry.
+
+## KubeCon Talks and Maintainer Sessions
```

Comment:
Add EMEA end-user session? CC: @martinjt

Reply:
yes, please, gimme the times & dates and I add them

Reply:
So, we haven't fixed that yet (unless I'm missing something), would it make sense to do it during the pavilion hours?

Reply:
I would guess so, probably worth checking at #otel-maintainers slack.